### ISSUE TITLE
Allow use of SSLKEYLOGFILE to examine DoH network captures

### ISSUE THREAD

**Issue by jasper-** [ameshkov/dnslookup#52]
I would like to show the inner workings of DoH to students.
For this I use wireshark and use the SSLKEYLOGFILE environment variable to store shared secrets of TLS sessions.
When using dnslookup after having set SSLKEYLOGFILE variable using export, the file does not get created.
I am running dnslookup v. 1.8.1-8619 - installed from the snap-store - on Ubuntu 22.04.2 LTS.

Would it be an option to make this work?

**Comment by grasstractor** [comment]
> First, this functionality should be added to [dnsproxy](https://github.com/AdguardTeam/dnsproxy) as dnslookup uses it under the hood.

I have the same question regarding DoQ. Do [dnsproxy](https://github.com/AdguardTeam/dnsproxy) and [quic-go](https://github.com/quic-go/quic-go) already have this functionality, or should they add it?"

**Comment by ameshkov** [comment]
quic-go seems to support it, dnsproxy not yet.

**Comment by grasstractor** [comment]
> quic-go seems to support it, dnsproxy not yet.

Yes, I found that quic-go has the functionality to support it. However, dnsproxy still needs to support it, or it will be too difficult for users to use it with dnslookup.
